### PR TITLE
Updates to CAP tutorials

### PR DESCRIPTION
**Pull request by slavipande**
Files are in the old v1 Tutorial Navigator syntax, which should be still supported, as shared by Joshua.

### CODE REVIEW

File: tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
 
-# Undeploy Your CAP Application from Kyma
-<!-- description --> This tutorial shows you how to undeploy your CAP application from Kyma.
-
 ## Prerequisites
  - [Assign a Role Collection to a User](btp-app-kyma-role-assignment)
 
 
-## You will learn
+## Details
```
Comment (MichaelCzcz) [inline tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md:17]: Please remove this line!

File: tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+### You will learn
```
Comment (MichaelCzcz) [inline tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md:18]: This should be: ## You will learn. Also, please move this section above ## Prerequisites

File: tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
  - How to undeploy your CAP application from Kyma
 
 
 ---
 
-### Undeploy your CAP application from Kyma
-
+[ACCORDION-BEGIN [Step 1: ](Undeploy your CAP application from Kyma)]
```
Comment (MichaelCzcz) [inline tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md:24]: Wrong Syntax.

File: tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md [hunk 4]
```diff
@@ -48,5 +46,6 @@ primary_tag: software-product-function>sap-cloud-application-programming-model
     kubectl delete namespace  risk-management
     ```
 
-
----
\ No newline at end of file
+[VALIDATE_1]
```
Comment (MichaelCzcz) [inline tutorials/btp-app-kyma-undeploy-cap-application/btp-app-kyma-undeploy-cap-application.md:49]: Remove this line. Validation is handled by rules.vr only

File: tutorials/btp-app-prepare-dev-environment-cap/btp-app-prepare-dev-environment-cap.md [hunk 5]
```diff
@@ -1,25 +1,24 @@
...
-# Prepare Your Development Environment for CAP
-<!-- description --> This tutorial shows you how to get started using VS Code as an editor create a directory for development, and start from an example branch.
-
 ## Prerequisites
  - (For Windows) You have to install [SQLite](https://sqlite.org/download.html) tools for Windows. Find the steps how to install it in the CAP documentation in section [How Do I Install SQLite](https://cap.cloud.sap/docs/advanced/troubleshooting#how-do-i-install-sqlite-on-windows).
  - (For macOS) You have to install Command-Line Tools for Xcode, cause some node modules need binary modules (`node-gyp`). There are two options to install them:
     - Using the command line: `xcode-select --install`
     - Downloading [Xcode](https://developer.apple.com/xcode/) (a login with your Apple ID and password is required). Search for `Command Line Tools for Xcode`.
 
+## Details
```
Comment (MichaelCzcz) [inline tutorials/btp-app-prepare-dev-environment-cap/btp-app-prepare-dev-environment-cap.md:19]: Remove this line

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
These tutorials have severe errors, because you have changed them back to the deprecated v1 parser syntax. I have only pointed out the first issues that I have noticed. You should have received an email with details about the new syntax. I you have not received it, please get in touch with Iwona.

**Comment by slavipande** [comment]
> These tutorials have severe errors, because you have changed them back to the deprecated v1 parser syntax. I have only pointed out the first issues that I have noticed. You should have received an email with details about the new syntax. I you have not received it, please get in touch with Iwona.

Hi @MichaelCzcz ,

Looks like there's a misunderstanding in our team's communication with @jmmargo. My colleague @manjuX had checked with Joshua to confirm if the v1 syntax would still be supported. We're authoring our tutorials in MkDocs and have a tool that converts our syntax to v1. We did not update our tool based on this understanding that v1 will still be supported. 
I guess we can follow up on this once Joshua is back from his OOO.

Best regards,
Svetoslav

**Comment by MichaelCzcz** [comment]
@slavipande I will discuss this with Joshua and get back to you.

**Comment by jmmargo** [comment]
The v1 parser is still active.  A tutorial may be authored in v1 or v2; we ask that they are not mixed.

Thanks,
Joshua

**Comment by jmmargo** [comment]
Yes, you may still author in v1 syntax.  Please be sure it is consistent.

-Joshua

**Pull request merged**